receptorctl/receptorctl/cli.py

```diff
@@ -114,7 +114,7 @@ def status(ctx):
             elif ad['ConnType'] == 2:
                 conn_type = 'StreamTLS'
             print(
-                f"{ad['NodeID']:<{longest_node}} {ad['Service']:<9} {conn_type:<10} {time:%Y-%m-%d %H:%M:%S} {(ad['Tags'] or '-'):<16}",
+                f"{ad['NodeID']:<{longest_node}} {ad['Service']:<9} {conn_type:<10} {time:%Y-%m-%d %H:%M:%S} {'-' if (ad['Tags'] is None) else str(ad['Tags']):<16}",
```

Comment:
does this if conditional honor the :<16 at the end? 

best way to test is to put some more characters after like `{'-' if (ad['Tags'] is None) else str(ad['Tags']):<16} here is more"` and make sure you still get 16 spaces after Tags entry